pycc/cclambda.py

```diff
@@ -204,6 +304,73 @@ def residuals(self, F, t1, t2, l1, l2):
         if isinstance(r1, torch.Tensor):
             del Goo, Gvv, Hoo, Hvv, Hov, Hovvo, Hovov, Hvvvo, Hovoo, Hvovv, Hooov
 
+        if self.ccwfn.model == 'CC3':                                                   
```

Comment:
I'm confused because this entire section appears to be redundant with the code above.  Am I missing something?

Reply:
This is in the residual function that's used only in the RT propagation (not in the iterations for the ground state calculation). So I copied the changes from solve_lambda()